### DISCUSSION TITLE
Script editor in Washington release

### DISCUSSION THREAD

**Discussion by Scumworks** [arnoudkooi/SN-Utils#543] [Q&A] [answered]
Greetings :)

We recently upgraded to Washington, which comes with some changes to the code editor. Whenever I access background script, it loads "modern". Though its an improvement for the build-in editor, its a real downgrade vs sn-utils. Is there a way to enable it again?

brgds

**Comment by vedang444** [comment]
I just noticed that as well.
Here is what I tried: In the SNUtils setting, I disabled the "Enhance Background Script Editor" setting, reloaded the page, and enabled it again. That worked for me.
Once there, you can click on 'Old' in the background script.

<img width="789" alt="image" src="https://github.com/user-attachments/assets/481b3724-ff40-4e7e-90a4-44c11acd17f8">

**Reply by Scumworks** [reply]
Hi vedang444, thanks for the answer. Hmm doesn't quite seem to work in my browser. When I reload, without or without "enhance background... " nothing seems to have changed.

**Comment by arnoudkooi** [comment, marked as answer]
Indeed, the Old link is a workaround. You can paste below in console to prevent the redirect to modern as well:
`localStorage.setItem(“snuFamily”, JSON.stringify({“family”:”dontredirect”,”checked”:”2024-01-01”,”override”:true}))`

You probably have an issue with the new modern script editor, that is known at support under KB1641470.
I attached the UI Macro that fixes it (rename .txt to .xml). But you may want to raise a case, if you prefer that.

[sys_ui_macro.txt](https://github.com/user-attachments/files/17290185/sys_ui_macro.txt)

**Reply by Scumworks** [reply]
Hi arnoudkooi, thanks for the support. Setting local storage did the trick, I'm back on track. You're somewhat of a legend in our team, can't imagine working without out the plugin. Thanks for all the hard work - if we cross paths, drinks are on me.

Brgds

**Reply by arnoudkooi** [reply]
Nice, but who is the mysterious person behind Scumworks 🤔?

**Reply by Scumworks** [reply]
Not sure how much I should disclose in a public forum 😄, but if you ever find yourself in Denmark 🍻.

Cheers
